MatrixSDK/Crypto/CryptoMachine/MXCryptoMachine.swift

```diff
@@ -562,7 +562,9 @@ extension MXCryptoMachine: MXCryptoRoomEventDecrypting {
             handleVerificationEvents: false,
             // The app does not use strict shields by default, in the future this will become configurable
             // per room.
-            strictShields: false
+            strictShields: false,
+            // TODO: Not sure what to put here
+            decryptionSettings: .init(senderDeviceTrustRequirement: .crossSigned)
```

Comment:
should be `.crossSigned` or `.crossSignedOrLegacy` ?

Reply:
should be `.untrusted`, to maintain the previous behaviour

Reply:
or `.unverified` maybe?